prisma/schema.prisma

```diff
@@ -87,15 +93,15 @@ model Story {
 
   published Boolean
 
-  staffPick Boolean
-
   thumbnailUrl String
   coverCaption String @default("")
 
-  createdAt   DateTime
-  publishedAt DateTime
-  updatedAt   DateTime
-  quizzes     Quiz[]
+  createdAt       DateTime
+  publishedAt     DateTime
+  updatedAt       DateTime
+  quizzes         Quiz[]
+  staffPick       StaffPick?
```

Comment:
Just to make sure I'm understanding this correctly - is staffPick the current "staff pick status" and StaffPickRecord the history listed out in full? So if staffPick does not exist, the story is not currently a staff pick, but if it exists it is currently a staff pick, regardless of staffPickRecord?

Reply:
Yes, that is correct.